currency_rate_date_check/res_config.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Vauxoo (https://www.vauxoo.com) <info@vauxoo.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from openerp import fields, models, api
+
+
+class BaseConfigSettings(models.TransientModel):
+    _inherit = 'base.config.settings'
+
+    activate_currency_rate_max_delta = fields.Boolean(
+        default=False, string='Activate check of the max date rate',
+        help="This must be true to execute the validation of Max Time Delta "
+        "in Days for Currency Rates")
+
+    @api.model
+    def _default_company(self):
```

Comment:
Is not there this method in the inherited model?

Reply:
Did you mean in the `base.config.settings` model?

Reply:
yes